Marlin/src/module/motion.cpp

```diff
@@ -1502,8 +1502,8 @@ void set_axis_not_trusted(const AxisEnum axis) {
     // Skip to next if target position is behind current. So it only moves away from endstop.
     if (phaseDelta < 0) phaseDelta += 1024;
 
-    // Convert tmc µsteps(phase) to whole Marlin µsteps to mm to effector backout direction
-    const float mmDelta = int16_t(phaseDelta / phasePerUStep) * planner.steps_to_mm[axis] * effectorBackoutDir ;
+    // Convert tmc µsteps(phase) to whole Marlin µsteps to effector to backout direction to mm 
+    const float mmDelta = int16_t(phaseDelta / phasePerUStep) * effectorBackoutDir * planner.steps_to_mm[axis];
```

Comment:
Don't you lose precision dividing integers?

Reply:
It is very very much intentional, not every phase is addressable, we want to consistently stop before we overflow into the next marlin microstep. 
For instance on a system running 16 marlin microsteps per stepper full step advances 16 tmc microsteps per marlin microstep, all good there but the driver is initialized on phase 8, so with a single marlin microstep you get to phase 24, making say phase 16 unreachable.